### PR TITLE
Add Indexer Watcher

### PR DESCRIPTION
**Pull request by majecty**
Not tested yet.

### CODE REVIEW

File: indexer-watcher/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "codechain-indexer-watcher",
```
Comment (sgkim126) [inline indexer-watcher/package.json]: Please remove 'codechain-' prefix as we discussed.

File: indexer-watcher/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+                    new Notifications.IndexerSyncTooSlow(
+                        status.indexerBestBlockNumber,
+                        status.codechainBestBlockNumber,
+                    ),
+                    targetEmail,
+                );
+                return;
+            }
+
+            if (indexerWasSlow === "following" && !indexerIsSlow) {
```
Comment (sgkim126) [inline indexer-watcher/src/index.ts]: How about removing this block?

File: indexer-watcher/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+                indexerWasSlow = "following";
+                return;
+            }
+
+            if (indexerWasSlow === "slow" && indexerIsSlow) {
```
Comment (sgkim126) [inline indexer-watcher/src/index.ts]: ditto

File: indexer-watcher/src/Notification.ts [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+export class DailyReport implements Notification {
+    public readonly title: string;
+    public readonly content: string;
+    public readonly level = "info";
+    public readonly date = new Date();
+
+    public constructor() {
+        const prefix = `[${this.level}][${networkId}][indexer-watcher]`;
+        this.title = `${prefix} is working`;
+        this.content = "The Indexer Watcher is working without problem.";
```
Comment (sgkim126) [inline indexer-watcher/src/Notification.ts]: How about reporting the starting block and the last block of the day?

File: indexer-watcher/src/index.ts (hunk 2 above)
Comment (majecty) [reply indexer-watcher/src/index.ts]: I removed the empty blocks.

File: indexer-watcher/src/Notification.ts (hunk 4 above)
Comment (majecty) [reply indexer-watcher/src/Notification.ts]: I added the starting and last block number in the message.

**Comment by sgkim126** [comment]
Please update travis.yml

**Comment by majecty** [comment]
@sgkim126 I updated travis.yml

File: .travis.yml [hunk 5]
```diff
@@ -105,6 +105,19 @@ jobs:
           script:
               - yarn lint
               - yarn build
+        - name: indexer-watcher
+          os: linux
+          before_install:
+              - CHANGED=$(.travis/check-change '^fee-monitor/|^\.travis.yml$|\.editorconfig$') && if [[ "$CHANGED" = "unchanged" ]]; then exit 0; fi
```
Comment (sgkim126) [inline .travis.yml]:
```
              - CHANGED=$(.travis/check-change '^indexer-watcher/|^\.travis.yml$|\.editorconfig$') && if [[ "$CHANGED" = "unchanged" ]]; then exit 0; fi
```

Comment (majecty) [reply .travis.yml]: I fixed it. Thanks!

**Pull request merged**